hoomd/md/pytest/test_potential.py

```diff
@@ -424,6 +454,45 @@ def _valid_params(particle_types=['A', 'B']):
                                         dict(zip(combos,
                                                  dlvo_valid_param_dicts)),
                                         {}))
+
+    tersoff_arg_dict = {
+            'cutoff_thickness': [0.1, 0.5, 1.0],
```

Comment:
Regarding a test that randomly throws particles out of the box, I suggest changing the test to just run 1 step. The purpose of a "run" test like this is to ensure that the kernel executes and does something. Ensuring a stable simulation will require a proper selection of step size, potential parameters, and the initial condition which is challenging to accomplish in this fully parameterized fashion.

If the simulation is still unstable enough to through particles out of the box in just 1 step then we either need to use more appropriate parameters for the potential or consider dropping that test entirely.